Jenkinsfile

```diff
@@ -36,20 +42,96 @@ pipeline {
                   BRANCH=""
                 fi
 
-                # This means we'll use the master branch of our other repos
-                export RELEASE="git:master"
                 cd $WORKSPACE/deploy/ansible
 
-                # We restore a backup to ensure upgrades work
-                export RESTORE_BACKUP="$WORKSPACE/examples/schema-v0_3_3.tgz"
 
-                # The actual job
+
+                # Install the current pypi release
+                export CLOUD="localhost"
+                export FLOATING_IP_BLOCK="10.0.0.0/24"
+                export KSM_ENABLED="0"
+                ./deployandtest.sh
+
+                # Log what versions we are running
+                echo "============================================================"
+                pip list | grep shakenfist
+                echo "============================================================"
+          '''
+        }
+      }
+    stage('Create sample infrastructure') {
```

Comment:
Maybe start two of everything? Push a little harder on the upgrade script.

Reply:
Done.